presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -647,24 +660,38 @@ protected Void visitCreateTable(CreateTable node, Integer indent)
             if (node.isNotExists()) {
                 builder.append("IF NOT EXISTS ");
             }
-            builder.append(node.getName())
-                    .append(" (");
+            String tableName = node.getName().getOriginalParts().stream()
```

Comment:
Add another version of `formatName()` that takes a `QualifiedName` and formats it correctly with escaping